lab1b.md

```diff
@@ -205,187 +204,97 @@ NAME                                   STATUS    ROLES     AGE       VERSION
 gke-name-default-pool-eceef152-qjmt   Ready     <none>    1h        v1.10.7-gke.2
 ```
 
-## Configure a registry - The Docker Hub
-
-Sign up for a Docker Hub account. The [Docker Hub](https://hub.docker.com) allows you to publish your Docker images on the Internet for use on multi-node clusters or to share with the wider community. We will be using the Docker Hub to publish our functions during the workshop.
-
-You can sign up here: [Docker Hub](https://hub.docker.com)
-
-Open a Terminal or Git Bash window and log into the Docker Hub using the username you signed up for above.
-
-```
-$ docker login
-```
-
-> Note: Tip from community - if you get an error while trying to run this command on a Windows machine, then click on the Docker for Windows icon in the taskbar and log into Docker there instead "Sign in / Create Docker ID".
-
 ## Deploy OpenFaaS
 
 The instructions for deploying OpenFaaS change from time to time as we strive to make this even easier.
 
-### Install the helm CLI/client
+### Install OpenFaaS
 
-Instructions for latest Helm install
+There is a new tool called `k3sup` which can install helm charts, including OpenFaaS. This makes the installation procedure much faster.
 
-* On Linux and Mac/Darwin:
+#### Install with `k3sup`
 
-```sh
-curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
-```
+* Get k3sup
 
-* Or via Homebrew on Mac:
+For Windows:
 
 ```sh
-brew install kubernetes-helm
+curl -SLsf https://get.k3sup.dev/ | sh
 ```
-      
-On Windows [download the helm.exe file](https://helm.sh) and place it in $PATH or /usr/bin/.
-
-### Install tiller
 
-* Create RBAC permissions for tiller
+For MacOS / Linux:
 
 ```sh
-kubectl -n kube-system create sa tiller \
-  && kubectl create clusterrolebinding tiller \
-  --clusterrole cluster-admin \
-  --serviceaccount=kube-system:tiller
+curl -SLsf https://get.k3sup.dev/ | sudo sh
 ```
 
-* Install the server-side Tiller component on your cluster
+* Install the OpenFaaS app
 
-```sh
-helm init --skip-refresh --upgrade --service-account tiller
-```
-
-### Install OpenFaaS with helm
-
-* Install the OpenFaaS helm chart
-
-We first create two namespaces `openfaas` and `openfaas-fn`:
+If you're using a managed cloud Kubernetes service which supplies LoadBalancers, then run the following:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml
+k3sup app install openfaas --loadbalancer
 ```
 
-Now add the helm chart repo for the project:
+If you're using a local Kubernetes cluster or a VM, then run:
 
 ```sh
-helm repo add openfaas https://openfaas.github.io/faas-netes/
+k3sup app install openfaas
```

Comment:
I think you could even unify this into the pervious block by using

```
k3sup app install openfaas --loadbalancer=true/false
```
and tell the user to use false when running in a local cluster.  or inversely to use true when deploying to a cloud

Reply:
The default is false

Reply:
Added a note as suggested.